UI/obs-app.cpp

```diff
@@ -3176,6 +3217,47 @@ static void upgrade_settings(void)
 	os_closedir(dir);
 }
 
+static const char *sentinelContents = "This file intentionally left blank.";
+
+static void check_safe_mode_sentinel(void)
+{
+#ifndef NDEBUG
```

Comment:
I wonder if it would be more efficient to guard the _function call_ behind that instead of creating an immediately returning function, though I would also suspect that modern compilers optimise that away anyway?